### PR TITLE
catbox: homepage inclusion

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

Added `homepage` key to `package.yml` (Part of #411)

**Test Plan**

Verified the added homepage loads information about the package

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by malfisya** [comment]
what does the warning mean?

File: packages/c/catbox/package.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
 name       : catbox
 version    : 1.7.0
-release    : 6
+release    : 7
 source     :
     - https://files.pythonhosted.org/packages/source/c/catbox/catbox-1.7.0.tar.gz : f0f066048679a9f5a00b9ccd83c06fb0b98d5d5f27dab1ba0c6ef325e71db109
+homepage   : https://pypi.org/project/catbox/
 license    : GPL-2.0-or-later
 component  : system.devel
```
Comment (ReillyBrogan) [inline packages/c/catbox/package.yml:8]: `system.devel` is not a good component for this, can you figure out a better one?

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]

**Comment by ReillyBrogan** [comment]
> what does the warning mean?

Well, catbox is in system.devel which is installed into all solbuild build environments. It really shouldn't be there. `system.devel` is sort of like `system.base` in that all dependencies of `system.devel` packages also need to be in `system.devel` or `system.base`

File: packages/c/catbox/package.yml (hunk 1 above)
Comment (malfisya) [reply packages/c/catbox/package.yml:8]: what about `programming.library` ,`programming.devel`, or anything on `programming` ?

Comment (ReillyBrogan) [reply packages/c/catbox/package.yml:8]: programming.devel maybe?

File: packages/c/catbox/package.yml [hunk 2]
```diff
@@ -1,14 +1,16 @@
 name       : catbox
 version    : 1.7.0
-release    : 6
+release    : 7
 source     :
     - https://files.pythonhosted.org/packages/source/c/catbox/catbox-1.7.0.tar.gz : f0f066048679a9f5a00b9ccd83c06fb0b98d5d5f27dab1ba0c6ef325e71db109
+homepage   : https://pypi.org/project/catbox/
 license    : GPL-2.0-or-later
-component  : system.devel
+component  : programming.devel
```
Comment (davidjharder) [inline packages/c/catbox/package.yml:8]: I think `programming.tools` would be better, `devel` is for libraries

**Review by davidjharder — CHANGES_REQUESTED** [review]

**Review by davidjharder — APPROVED** [review]
⚽

**Pull request merged**